baker/GrapherBaker.tsx

```diff
@@ -436,29 +405,25 @@ export const bakeAllPublishedChartsVariableDataAndMetadata = async (
     variableIds: number[],
     checksumsDir: string
 ) => {
-    await fs.mkdirp(`${bakedSiteDir}${GRAPHER_VARIABLES_ROUTE}`)
-    await fs.mkdirp(`${bakedSiteDir}${GRAPHER_VARIABLE_DATA_ROUTE}`)
-    await fs.mkdirp(`${bakedSiteDir}${GRAPHER_VARIABLE_METADATA_ROUTE}`)
     await fs.mkdirp(checksumsDir)
 
     const progressBar = new ProgressBar(
-        "bake variable data/metadata json [:bar] :current/:total :elapseds :rate/s :etas :name\n",
+        "check existence of variable in Data API [:bar] :current/:total :elapseds :rate/s :etas :name\n",
         {
             width: 20,
             total: variableIds.length + 1,
         }
     )
 
-    const jobs: BakeVariableDataArguments[] = variableIds.map((variableId) => ({
-        bakedSiteDir,
-        variableId,
-        checksumsDir,
-    }))
-
+    // NOTE: we don't bake data, just make sure it exists on S3
     await Promise.all(
-        jobs.map(async (job) => {
-            await bakeVariableData(job)
-            progressBar.tick({ name: `variableid ${job.variableId}` })
+        variableIds.map(async (variableId) => {
+            await bakeVariableData({
```

Comment:
Not sure since in your testing I assume this ran fine; but this is making a lot of requests at the same time, so same chunking or limiting could be helpful to prevent errors in the future.

Reply:
I haven't run into any problems with this yet. It's making HEAD requests, so I guess Cloudflare doesn't have a problem with it.

Anyway, if we run into any issues, I'll start chunking it. It could also be turned off if Bugsnag can monitor 403s for us.